openrouteservice-api-tests/conf/app.config.test

```diff
@@ -52,6 +52,8 @@
             maximum_segment_distance_with_dynamic_weights: 50000,
             maximum_waypoints: 50,
             maximum_snapping_radius: 350,
+            maximum_avoid_polygon_area: 2.5,
+            maximum_avoid_polygon_extent: 1.5,
```

Comment:
What's the units?

Reply:
square kilometers / kilometers, and yes, the extent checks both x and y.

Reply:
I'd rather stay with meters as units for consistency's sake. All limiting distance/area values are in meters.

Reply:
ok, changed.